Filebeat mqtt add cleansession (#35806)
elastic/beats: filebeat/docs/inputs/input-mqtt.asciidoc

A client password used for authentication provided on the application level by the MQTT protocol.

===== `clean_session`

> The clean_session flag indicates whether the client wants to establish a persistent session with the broker. 

Reviewer: Sorry, missed a spot in the review.

```suggestion
The `clean_session` flag indicates whether the client wants to establish a persistent session with the broker. 
```

> The default is `false`.

Reviewer: I thought the default was `true`?

Author: Your write, sorry for this mistake.

> When clean_session is set to false, considered a persistent session, the broker stores all subscriptions for 

Reviewer: ```suggestion
When `clean_session` is set to false, the session is considered to be persistent. The broker stores all subscriptions for 
```

the client and all missed messages for the client that subscribed with a Quality of Service (QoS) level 1 or 2. 

> In contrast, when CleanSession is set to true, the broker doesn’t retain any information for the client 

Reviewer: Consistency.

```suggestion
In contrast, when `clean_session` is set to true, the broker doesn’t retain any information for the client 
```
